src/engine/V3Eval.v3

```diff
@@ -609,6 +785,36 @@ def I8_POPCNT(a: u8) -> u8 {
 	return count;
 }
 // Adapters
+def u8_bool_binop(a: u8, b: u8, f: (u8, u8) -> bool) -> u8 {  // Adapts a unsigned bool binop to a u8 binop
```

Comment:
What about `do_bb_z` as a name here? We haven't used an abbreviation for bytes yet, but we could use `b` for byte, `w` for u16, and `q` for v128. I use `z` for boolean in, e.g. Virgil's `lib/util`.

Reply:
Yeah the naming of these helpers did bother me a while. Thanks for the suggestion and I'll push a new commit for this.

Reply:
Just to double-check: for v128, should we use `dq` as what we did in the x86_64Interpreter? And, accordingly, use `d` for u32 and `q` for u64?

Reply:
I would propose this:

```
b = byte = u8
h = u16
i = i32
u = u32
l = i64
w = u64
f = float
d = double
q = v128
```

Reply:
Yes this looks clean. Thanks! Pushing a new commit soon.